tools/testgrid/testgrid.go

```diff
@@ -49,6 +49,22 @@ type TestSuite struct {
 	TestCases []TestCase `xml:"testcase"`
 }
 
+// GetArtifactsDir gets the aritfacts directory where we should put the artifacts.
+// By default, it will look at the env var ARTIFACTS.
+func GetArtifactsDir() string {
+	dir := os.Getenv("ARTIFACTS")
+	if dir == "" {
+		return "./artifacts"
```

Comment:
For the sake of debugging, add a log here saying we're using the hardcoded `./artifacts` dir.